src/include/sof/trace.h

```diff
@@ -35,30 +35,35 @@
  * when main trace is not available
  */
 #define TRACE_BOOT_START		0x1000
-#define TRACE_BOOT_ARCH		0x2000
+#define TRACE_BOOT_ARCH			0x2000
 #define TRACE_BOOT_SYS			0x3000
 #define TRACE_BOOT_PLATFORM		0x4000
 
 /* system specific codes */
-#define TRACE_BOOT_SYS_CPU_FREQ		(TRACE_BOOT_SYS + 0x200)
-#define TRACE_BOOT_SYS_HEAP		(TRACE_BOOT_SYS + 0x300)
-#define TRACE_BOOT_SYS_NOTE		(TRACE_BOOT_SYS + 0x400)
-#define TRACE_BOOT_SYS_SCHED		(TRACE_BOOT_SYS + 0x500)
-#define TRACE_BOOT_SYS_POWER		(TRACE_BOOT_SYS + 0x600)
+#define TRACE_BOOT_SYS_HEAP		(TRACE_BOOT_SYS + 0x100)
+#define TRACE_BOOT_SYS_TRACES		(TRACE_BOOT_SYS + 0x200)
+#define TRACE_BOOT_SYS_NOTIFIER		(TRACE_BOOT_SYS + 0x300)
+#define TRACE_BOOT_SYS_POWER		(TRACE_BOOT_SYS + 0x400)
 
 /* platform/device specific codes */
 #define TRACE_BOOT_PLATFORM_ENTRY	(TRACE_BOOT_PLATFORM + 0x100)
-#define TRACE_BOOT_PLATFORM_MBOX	(TRACE_BOOT_PLATFORM + 0x110)
-#define TRACE_BOOT_PLATFORM_SHIM	(TRACE_BOOT_PLATFORM + 0x120)
-#define TRACE_BOOT_PLATFORM_PMC		(TRACE_BOOT_PLATFORM + 0x130)
-#define TRACE_BOOT_PLATFORM_TIMER	(TRACE_BOOT_PLATFORM + 0x140)
-#define TRACE_BOOT_PLATFORM_CLOCK	(TRACE_BOOT_PLATFORM + 0x150)
-#define TRACE_BOOT_PLATFORM_SSP_FREQ	(TRACE_BOOT_PLATFORM + 0x160)
-#define TRACE_BOOT_PLATFORM_IPC		(TRACE_BOOT_PLATFORM + 0x170)
-#define TRACE_BOOT_PLATFORM_DMA		(TRACE_BOOT_PLATFORM + 0x180)
-#define TRACE_BOOT_PLATFORM_SSP		(TRACE_BOOT_PLATFORM + 0x190)
-#define TRACE_BOOT_PLATFORM_DMIC	(TRACE_BOOT_PLATFORM + 0x1a0)
-#define TRACE_BOOT_PLATFORM_IDC		(TRACE_BOOT_PLATFORM + 0x1b0)
+#define TRACE_BOOT_PLATFORM_IRQ		(TRACE_BOOT_PLATFORM + 0x110)
```

Comment:
Not sure how this works, but I'm wondering if it is really OK to change the trace point numbers? Are there any tools depending on this exact numbers?

Reply:
They are internal and not used by any tool, so it's OK.